editor/src/clj/editor/app_view.clj

```diff
@@ -2655,7 +2661,7 @@ If you do not specifically require different script states, consider changing th
 (defn- gen-tooltip [workspace project app-view resource]
   (when (resource/loaded? resource)
     (let [resource-type (resource/resource-type resource)
-          view-type (or (first (:view-types resource-type))
+          view-type (or (first (view-types resource))
```

Comment:
I would prefer not to use the more sophisticated view-type check here, since it reads part of the file to determine if it is binary or not. The `gen-tooltip` function appears to be called for every resource presented in asset picker dialogs, so introducing a disk read per resource could slow things down. The `:code` view does not specify `:make-preview-fn` anyway.